Meadowlark-js/packages/meadowlark-core/package.json

```
@@ -0,0 +1,82 @@
+{
+  "name": "@edfi/meadowlark-core",
+  "version": "0.2.0-dev.0",
+  "description": "Meadowlark core functionality",
+  "license": "SEE LICENSE IN LICENSE.md",
```

Would it makes sense to change this for Meadowlark?
```suggestion
  "license": "Apache-2.0",
```

---

Done